### PR TITLE
test: updated tests for checkboxgroup

### PR DESCRIPTION
**Pull request by NandanAnantharamu**
Added tests for checkbox group
https://github.com/appsmithorg/TestSmith/issues/2479

### CODE REVIEW

**Comment by NandanAnantharamu** [comment]
ci-test-limit

File: app/client/cypress/e2e/Regression/ClientSide/Widgets/Checkbox/CheckBoxGroupTest.ts [hunk 1]
```diff
@@ -0,0 +1,299 @@
...
+    );
+    propPane.ToggleJSMode("borderradius", true);
+    propPane.UpdatePropertyFieldValue("Border radius", "0.375rem");
+    agHelper.AssertAttribute(
+      `${propPane._checkbox} input`,
+      "borderradius",
+      "0.375rem",
+    );
+  });
+});
```
Comment (sharanya-appsmith) [inline app/client/cypress/e2e/Regression/ClientSide/Widgets/Checkbox/CheckBoxGroupTest.ts:299]: can we also add cases for
 1. Using JS 
 2. Using binding
 3. on Success and on failure in Events (i.e., callbacks)

**Review by sharanya-appsmith — DISMISSED** [review]

**Review by sharanya-appsmith — APPROVED** [review]

**Pull request merged**